tests/storage/snapshot.go

```diff
@@ -1068,6 +1061,12 @@ var _ = SIGDescribe("VirtualMachineSnapshot Tests", func() {
 				Expect(err).ToNot(HaveOccurred())
 
 				origStatus := ss.Status
+				emptyTime := metav1.Time{}
```

Comment:
Please extract the clear code to a method `clearConditionTimestamps`.

Reply:
done